### PR TITLE
SAR-8696 Update config to include BV check

### PR DESCRIPTION
**Pull request by woollett-r**

### CODE REVIEW

File: app/uk/gov/hmrc/soletraderidentificationfrontend/testonly/views/test_create_individual_journey.scala.html [hunk 1]
```diff
@@ -84,6 +85,12 @@
             classes = "govuk-input--width-20"
         )
 
+        @inputHidden(
+            id = "businessVerificationCheck",
+            name = "businessVerificationCheck",
+            value = form("businessVerificationCheck").value.getOrElse("true")
+        )
+
```
Comment (HarrNM) [inline app/uk/gov/hmrc/soletraderidentificationfrontend/testonly/views/test_create_individual_journey.scala.html:93]: Do we need this on the individual creation page? Why is it hidden?

File: app/uk/gov/hmrc/soletraderidentificationfrontend/testonly/views/test_create_sole_trader_journey.scala.html [hunk 2]
```diff
@@ -84,6 +85,25 @@
             classes = "govuk-input--width-20"
         )
 
+       @govukCheckboxes(Checkboxes(
+         fieldset = Some(Fieldset(
+           legend = Some(Legend(
+             content = Text("Business verification check"),
+             classes = "govuk-fieldset__legend--m",
```
Comment (HarrNM) [inline app/uk/gov/hmrc/soletraderidentificationfrontend/testonly/views/test_create_sole_trader_journey.scala.html:92]: This causes the heading to look different than the others on this page. Doesn't really matter but could we change it to match? Will probably just need to give it the same classes as the others have `govuk-label govuk-label govuk-!-font-weight-bold`

File: app/uk/gov/hmrc/soletraderidentificationfrontend/testonly/views/test_create_individual_journey.scala.html (hunk 1 above)
Comment (woollett-r) [reply app/uk/gov/hmrc/soletraderidentificationfrontend/testonly/views/test_create_individual_journey.scala.html:93]: The business verification deactivation is not relevant to the individual journey so I don't think we need the checkbox. However, to keep the form happy I thought it a good idea to submit something. Further, if we submit nothing it is likely the business verification flag will be set to "false". As the default value is actually "true" I think it is better to use "true".

File: app/uk/gov/hmrc/soletraderidentificationfrontend/testonly/views/test_create_sole_trader_journey.scala.html (hunk 2 above)
Comment (woollett-r) [reply app/uk/gov/hmrc/soletraderidentificationfrontend/testonly/views/test_create_sole_trader_journey.scala.html:92]: I'll change the classes and have a look.

**Pull request merged**